sys/include/xtimer.h

```diff
@@ -445,10 +445,17 @@ int xtimer_mutex_lock_timeout(mutex_t *mutex, uint64_t us);
  * This function will set THREAD_FLAG_TIMEOUT on the current thread after @p
  * timeout usec have passed.
  *
- * @param[in]   t       timer struct to use
+ * @param[in]   timer   timer struct to use
```

Comment:
why renaming this, when its actual implementation still uses `t`?

Reply:
Ping @kaspar030?